crates/utils/src/rate_limit/mod.rs

```diff
@@ -190,7 +192,7 @@ where
     self.service.poll_ready(cx)
   }
 
-  fn call(&mut self, req: S::Request) -> Self::Future {
+  fn call(&mut self, req: Self::Request) -> Self::Future {
```

Comment:
@asonix I upgraded to actix-web `4.0.0-beta.8` but actix changed how they do transforms, and I'm pretty lost on how to get this file compiling again, I could really use your help.

[Here's their docs on Transform](https://docs.rs/actix-web/4.0.0-beta.8/actix_web/dev/trait.Transform.html)

Reply:
&must Self turns into &Self, Request moves from an associated type to a generic on Service and Transform. It's pretty mechanical

Reply:
With what trait bounds? I can't do `let ip_addr = get_ip(&req.connection_info());` on a generic.

Reply:
Instead of being generic over the Request type, only implement the trait over ServiceRequest

Reply:
Cool, thx!